app/scripts/modules/core/pipeline/config/stages/createLoadBalancer/createLoadBalancerStage.html

```diff
@@ -0,0 +1,55 @@
+
+<div ng-controller="createLoadBalancerStageCtrl as createlbStageCtrl" class="well well-sm clearfix" ng-if="!pipeline.strategy">
```

Comment:
camelCasing is off on the controllerAs - `createlbStageCtrl` should be `createLbStageCtrl`. Or you could change it to something generic and short like `vm` or `ctrl.